src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -116,7 +116,16 @@ public override async Task<bool> Execute()
 
                 Logger.DebugLocalized(nameof(Resources.RetrievingManifest_Message), this.Id);
 
-                string exactId = await client.FindPackageId(this.Id);
+                string exactId = string.Empty;

```

Comment:
Why empty instead of null?

Reply:
Oops, changed to uninitialized. I think I thought I was working with a var